src/components/Form/InputWrapper.js

```diff
@@ -1,17 +1,20 @@
 import PropTypes from 'prop-types';
 import React, {forwardRef, useContext} from 'react';
+import refPropTypes from '@components/refPropTypes';
```

Comment:
We should not, please check out my comment to the previous change request of this type

Reply:
Apparently in this case lint throws an error for using it the other way, it might be that it's either `import refPropTypes from '@components/refPropTypes';` when it's not in the same directory or `import refPropTypes from './refPropTypes'` when it's in the same directory.

Reply:
This is correct as it was announced by the team on slack, one of the quirks of this system is as follows:

https://github.com/dword-design/eslint-plugin-import-alias/issues/133